[2023-08-30T07:40:43.171+0000] {taskinstance.py:1103} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: process_sale_data.check_file_exist manual__2023-08-30T07:40:41.616474+00:00 [queued]>
[2023-08-30T07:40:43.179+0000] {taskinstance.py:1103} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: process_sale_data.check_file_exist manual__2023-08-30T07:40:41.616474+00:00 [queued]>
[2023-08-30T07:40:43.179+0000] {taskinstance.py:1308} INFO - Starting attempt 1 of 1
[2023-08-30T07:40:43.190+0000] {taskinstance.py:1327} INFO - Executing <Task(PythonOperator): check_file_exist> on 2023-08-30 07:40:41.616474+00:00
[2023-08-30T07:40:43.194+0000] {standard_task_runner.py:57} INFO - Started process 2518 to run task
[2023-08-30T07:40:43.197+0000] {standard_task_runner.py:84} INFO - Running: ['***', 'tasks', 'run', 'process_sale_data', 'check_file_exist', 'manual__2023-08-30T07:40:41.616474+00:00', '--job-id', '50', '--raw', '--subdir', 'DAGS_FOLDER/dags.py', '--cfg-path', '/tmp/tmpd1croeht']
[2023-08-30T07:40:43.199+0000] {standard_task_runner.py:85} INFO - Job 50: Subtask check_file_exist
[2023-08-30T07:40:43.237+0000] {task_command.py:410} INFO - Running <TaskInstance: process_sale_data.check_file_exist manual__2023-08-30T07:40:41.616474+00:00 [running]> on host 888526fa4a36
[2023-08-30T07:40:43.305+0000] {taskinstance.py:1547} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='process_sale_data' AIRFLOW_CTX_TASK_ID='check_file_exist' AIRFLOW_CTX_EXECUTION_DATE='2023-08-30T07:40:41.616474+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2023-08-30T07:40:41.616474+00:00'
[2023-08-30T07:40:43.319+0000] {python.py:183} INFO - Done. Returned value was: None
[2023-08-30T07:40:43.328+0000] {taskinstance.py:1350} INFO - Marking task as SUCCESS. dag_id=process_sale_data, task_id=check_file_exist, execution_date=20230830T074041, start_date=20230830T074043, end_date=20230830T074043
[2023-08-30T07:40:43.370+0000] {local_task_job_runner.py:225} INFO - Task exited with return code 0
[2023-08-30T07:40:43.396+0000] {taskinstance.py:2653} INFO - 1 downstream tasks scheduled from follow-on schedule check
